modules/configuring-revision-timeout.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: PROCEDURE
+[id="configuring-revision-timeout_{context}"]
+= Configuring revision timeout
+
+You can configure the default number of seconds to use for the per-request revision timeout.
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} and Knative Serving.
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+
+.Procedure
+
+* To configure the revision timeout globally, set the `revision-timeout-seconds` field in the `KnativeServing` custom resource (CR):
```

Comment:
s/
* Choose the appropriate method to configure the revision timeout:
**  To configure the revision timeout globally, set the `revision-timeout-seconds`  ...
**  To configure the timeout based on the revision, set the `timeoutSeconds` .... 

or

* To configure the revision timeout, do any of the following tasks:
** Set the `revision-timeout-seconds`  ...
** Set the `timeoutSeconds` ....